examples/demo-apps/apple_ios/ExecuTorchDemo/ExecuTorchDemo.xcodeproj/project.pbxproj

```diff
@@ -633,7 +646,7 @@
 				INFOPLIST_KEY_UIRequiresFullScreen = YES;
 				INFOPLIST_KEY_UISupportedInterfaceOrientations = UIInterfaceOrientationPortrait;
 				MARKETING_VERSION = 1.0;
-				PRODUCT_BUNDLE_IDENTIFIER = org.pytorch.executorch.demo;
+				PRODUCT_BUNDLE_IDENTIFIER = org.pytorch.executorch.demo.test;
```

Comment:
@shoumikhin is more knowledgable to review changes in this file. Can you take a look @shoumikhin ?

Reply:
This change is intentional because I use the demo app to host the test suite.  As such, the bundle identifier must match the value set in the provisioning profile, `org.pytorch.executorch.demo.test`, to sign it.